lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -441,9 +441,16 @@ def publishing_api_has_links(links)
       #           ]
       #         }
       #       }
-      def publishing_api_has_expanded_links(links, with_drafts: true)
+      def publishing_api_has_expanded_links(links, with_drafts: true, generate: false)
         links = deep_transform_keys(links, &:to_sym)
-        query = with_drafts ? "" : "?with_drafts=false"
+        if generate && !with_drafts
```

Comment:
Would this be easier to do as publishing_api_has_linked_items (line 528) - i.e. create a hash of parameters and then supply it to stub_request as a` with(query: hash_of_parameters)`